djantic/main.py

```diff
@@ -2,6 +2,7 @@
 from enum import Enum
 from functools import reduce
 from itertools import chain
+from types import NoneType, UnionType
```

Comment:
`NoneType` exists from python 3.10, is it possible to use `None` instead? I wouldn't want to drop 3.8, and 3.9 yet. 

3.8 will be EOL this autumn so will be open to drop that version then.